homeassistant/components/samsungtv/config_flow.py

```diff
@@ -220,7 +220,7 @@ async def _async_get_and_check_device_info(self) -> bool:
             raise AbortFlow(RESULT_NOT_SUPPORTED)
         self._model = dev_info.get("modelName")
         name = dev_info.get("name")
-        self._name = name.replace("[TV] ", "") if name else device_type
+        self._name = name.removeprefix("[TV] ") if name else device_type
```

Comment:
Again slight behavior change if `[TV] ` is not at the beginning.

Reply:
06c53516a1f61735ffa4ac0aa80bf06475b55fab